src/platform/miralis.rs

```diff
@@ -4,46 +4,25 @@ use core::fmt;
 
 use log::Level;
 use miralis_abi::{failure, miralis_log_fmt, success};
-use spin::Mutex;
 
-use crate::config::{TARGET_FIRMWARE_ADDRESS, TARGET_PAYLOAD_ADDRESS};
-use crate::device::clint::{VirtClint, CLINT_SIZE};
-use crate::device::tester::{VirtTestDevice, TEST_DEVICE_SIZE};
+use crate::device::clint::CLINT_SIZE;
+use crate::device::tester::TEST_DEVICE_SIZE;
 use crate::device::VirtDevice;
-use crate::driver::clint::ClintDriver;
+use crate::platform::{Plat, VIRT_CLINT, VIRT_TEST_DEVICE};
 use crate::Platform;
 
-// —————————————————————————— Platform Parameters ——————————————————————————— //
-
-const MIRALIS_START_ADDR: usize = TARGET_FIRMWARE_ADDRESS;
-const FIRMWARE_START_ADDR: usize = TARGET_PAYLOAD_ADDRESS;
-const CLINT_BASE: usize = 0x2000000;
```

Comment:
We can factor-our the MIRALIS_START and FIRMWARE_START, but let's keep the clint address in the platform folder. It is clearer to keep all platform variables in their own file, even though there are some standard addressees for most platforms.